bayes_opt/helpers.py

```diff
@@ -59,6 +59,10 @@ def acq_max(ac, gp, y_max, bounds, random_state, n_warmup=100000, n_iter=250):
                        bounds=bounds,
                        method="L-BFGS-B")
 
+        # See if success
+        if res.success==False:
```

Comment:
Can you change to `if not res.success:`? This is more pythonic.

Reply:
OK, I have changed